habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -954,11 +954,11 @@ class SimulatorConfig(HabitatBaseConfig):
     # Rearrange agent grasping
     hold_thresh: float = 0.09
     grasp_impulse: float = 1000.0
-    agents: List[str] = field(default_factory=lambda: ["agent_0"])
-    agent_0: AgentConfig = AgentConfig()
-    agent_1: AgentConfig = (
-        MISSING  # temporary multi-agent rearrangement config fix
-    )
+    # we assume agent(s) to be set explicitly
+    agents: Dict[str, AgentConfig] = MISSING
+    # if the number of agents is greater than one,
+    # then agents_order has to be set explicitly too
```

Comment:
Maybe something like this:
```suggestion
    # Agents order that will be used by simulator step and trainer policy.
```

Reply:
agent_order is only used to know in which order to return the observations and accept the actions when using the environment API correct? 
In any case, we should explain here what the agent_order does for the user

Reply:
Defining agent order like this seems a bit of an overkill. Why can't the order of the agents defined in the YAML just be the order of the agents? That seems a lot more intuitive to me instead of having this foreign key. Why cant the agents key be a list of of agents instead?